examples/rpc/README.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+
+The `rpc-server` allows  running `ggml` backend on a remote host.
+The RPC backend communicates with one or several instances of `rpc-server` and offloads computations to them.
```

Comment:
Does it mean llama.cpp allow any number of RPC backend workers as long as they are properly connected with the leader? Or there's limitation/requirement on this configuration. Sorry for a late comment as I just saw this proposal recently.

Reply:
yeah, looks there's not limitation of how many workers can connect.